functions/index.js

```diff
@@ -599,4 +599,50 @@ function createSprintId(sprintNumber) {
     } else {
         return ("S" + sprintNumber);
     }
-}
\ No newline at end of file
+}
+
+exports.createNewUser = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        const user = request.body.data;
+        const Uid = user.uid;
+        const PhotoURL = user.photoURL;
+        const DisplayName = user.displayName;
+        const Email = user.email;
+        const PhoneNumber = user.phoneNumber;
+        const ProviderId = user.providerId;
+
+        db.collection("Users").doc(Uid).get().then((doc) => {
+                if (doc.exists) {
+                    const userData = db.collection("Users").doc(Uid).update({
+                        uid: Uid,
+                        photoURL: PhotoURL,
+                        displayName: DisplayName,
+                        email: Email,
+                        phoneNumber: PhoneNumber,
+                        providerId: ProviderId
+                    });
```

Comment:
we don't need to do anything if document already exists here

Reply:
You can fix this by reversing the condition, to if the document does not exists.

Reply:
It modifies the updated Phone Number ,and PhotoURL Bhaiya .If they modify in Google Account .